include/deal.II/fe/mapping_q_cache.h

```diff
@@ -111,6 +112,38 @@ class MappingQCache : public MappingQGeneric<dim, spacedim>
                const typename Triangulation<dim, spacedim>::cell_iterator &)>
                &compute_points_on_cell);
 
+  /**
+   * Initialize the data cache by computing the mapping support points for all
+   * cells (on all levels) of the given triangulation and a given @p mapping
+   * and transforming these points via the function @p transformation_function.
```

Comment:
Out of curiosity: This is a way to move points, but how are you addressing the fact that the additional move would also lead to a different Jacobian (and its determinant)?

Reply:
Indeed, the beauty of this class is that you only need to move points; since this is a polynomial mapping, that's all you need, and the normals, Jacobians, etc. are computed from the points (in terms of the manifold paper, we have stored the result of the point computation in section 3.2 with "new point" and don't need anything else).

Reply:
Ah, I see -- thanks for educating me!